src/Compilers/Core/Portable/InternalUtilities/EnumUtilties.cs

```diff
@@ -52,5 +52,20 @@ internal static ulong ConvertEnumUnderlyingTypeToUInt64(object value, SpecialTyp
         {
             return (T[])Enum.GetValues(typeof(T));
         }
+
+#if DEBUG
+        internal static bool ContainsAllValues<T>(int mask) where T : struct, IConvertible
```

Comment:
>ContainsAllValues [](start = 29, length = 17)

Should the test performed by this function be stronger? I.e. combine all the values and then compare the result to the ```mask```.

Reply:
Perhaps although this should be equivalent to the previous tests.

---
In reply to: [286258365](https://github.com/dotnet/roslyn/pull/35846#discussion_r286258365) [](ancestors = 286258365)